web/src/components/CategoryListItem.tsx

```diff
@@ -65,25 +42,39 @@ type CategoryListItemProps = {
 }
 
 const CategoryListItem = ({ category, subCategories }: CategoryListItemProps): ReactElement => {
-  const SubCategories = subCategories.map(subCategory => (
-    <SubCategory key={subCategory.path} dir='auto'>
-      <StyledLink to={subCategory.path}>
-        {!!subCategory.thumbnail && <CategoryThumbnail alt='' src={subCategory.thumbnail} />}
-        <SubCategoryCaption>{subCategory.title}</SubCategoryCaption>
-      </StyledLink>
-      <Divider />
-    </SubCategory>
-  ))
+  const SubCategories = subCategories.map(subCategory => {
+    const { path, thumbnail, title } = subCategory
+    return (
+      <SubCategoryListItem key={path}>
+        <StyledSubCategoryListItemButton component={Link} to={path}>
+          {!!thumbnail && (
+            <ListItemAvatar>
+              <Avatar src={thumbnail} variant='square' />
+            </ListItemAvatar>
+          )}
+          <ListItemText primary={<Typography variant='body1'>{title}</Typography>} />
+        </StyledSubCategoryListItemButton>
+      </SubCategoryListItem>
+    )
+  })
 
   return (
-    <Row>
-      <StyledLink dir='auto' to={category.path}>
-        {!!category.thumbnail && <CategoryThumbnail alt='' src={category.thumbnail} />}
-        <CategoryItemCaption>{category.title}</CategoryItemCaption>
-      </StyledLink>
-      <Divider />
-      <SubCategoriesContainer>{SubCategories}</SubCategoriesContainer>
-    </Row>
+    <StyledListItem disablePadding>
+      <ListItemButton component={Link} to={category.path} dir='auto'>
+        {!!category.thumbnail && (
+          <ListItemAvatar>
+            <Avatar src={category.thumbnail} variant='square' />
+          </ListItemAvatar>
+        )}
+        <ListItemText primary={<StyledTypography variant='title2'>{category.title}</StyledTypography>} />
+      </ListItemButton>
+      {SubCategories.length > 0 && (
+        <>
+          <Divider />
+          <StyledList isInset items={SubCategories} />
```

Comment:
🔧 Instead of adding a padding on the button and adding a custom `isInset` property, just add a padding to the whole list:
```suggestion
          <Stack paddingInlineStart={7}>
            <StyledList items={SubCategories} />
          </Stack>
```

Reply:
:thinking:  is there any advantage from using `Stack` here?

Reply:
You can also add it as a style to `StyledList`, doesnt matter. I'd just prefer that over passing it thorugh to the items and the divider.